css/style.css

```diff
@@ -607,54 +607,54 @@ ul {
   background-size: cover;
   padding: 0;
 }
-.whychooseus-overlay {
+.overlay {
   background: rgba(0, 0, 0, 0.7);
   background-size: cover;
   padding-top: 70px;
   padding-bottom: 80px;
 }
-.whychooseus-container {
+.container2 {
   background-size: cover;
   padding-right: 15px;
   padding-left: 15px;
   margin-right: auto;
   margin-left: auto;
 }
-.whychooseus-row {
+.r {
   display: flex;
   flex-wrap: wrap;
   margin: 40px;
   /* color: #ffffff; */
   color: #ffffff;
   letter-spacing: 0.1rem;
 }
-.whychooseus-block {
+.block {
   display: flex;
   width: 31.5%;
   margin: 10px;
 }
-.whychooseus-block i {
+.block i {
   color: #ffffff;
   padding-right: 8px;
   font-size: 25px;
 }
-.whychooseus-coloumn h3{
-  font-size: 1.1rem;
+.col h3 {
+  font-size: 1.22rem;
   /* font-size: 16px; */
   font-family: 'Montserrat', Arial, Helvetica, sans-serif;
   line-height: normal;
   margin-bottom: 5px;
-  font-weight: 550;
+  font-weight: 600;
   letter-spacing: 0.01rem;
 }
-.whychooseus-coloumn p {
+.col p {
   font-family: 'Montserrat', Arial, Helvetica, sans-serif;
   font-size: 13px;
-  font-size: 0.8rem;
-  line-height: 1.5;
+  font-size: 0.87rem;
+  line-height: 1.6;
   letter-spacing: 0.099rem;
 }
-.whychooseus-coloumn hr {
+.col hr {
   margin-top: 10px;
   width: 30%;

```

Comment:
Why you are changing "Why choose us section" ?

Remove your whole changes and just change what you want only.